### PR TITLE
installed ESLint and Prettier

### PR DESCRIPTION
**Pull request by nathzz30**
Review this implementation, in the .eslintrc file.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,17 +14,19 @@
     "@babel/core": "^7.5.5",
     "@babel/preset-env": "^7.5.5",
     "@babel/preset-react": "^7.0.0",
+    "@emotion/core": "^10.0.14",
```
Comment (csuriel) [inline package.json]: This shouldn't be listed on `devDependencies`. 
The same for `react` and `react-dom`.

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  "extends": ["airbnb", "prettier", "prettier/react"],
+  "plugins": ["prettier"],
+  "rules": {
+    "react/jsx-filename-extension": [
+      1,
+      {
+        "extensions": [".js", ".jsx"]
+      }
+    ],
+    "react/prop-types": 0,
```
Comment (csuriel) [inline .eslintrc]: Why is this disabled?

File: .eslintrc [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    "no-underscore-dangle": 0,
+    "import/imports-first": ["error", "absolute-first"],
+    "import/newline-after-import": "error"
+  },
+  "globals": {
```
Comment (csuriel) [inline .eslintrc]: Are these globals actually necessary for this project?

File: .eslintrc [hunk 4]
```diff
@@ -0,0 +1,26 @@
+{
+  "extends": ["airbnb", "prettier", "prettier/react"],
+  "plugins": ["prettier"],
+  "rules": {
+    "react/jsx-filename-extension": [
```
Comment (csuriel) [inline .eslintrc]: Unnecessary

File: package.json [hunk 5]
```diff
@@ -5,7 +5,15 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "dev": "webpack-dev-server --mode development ./src/index.js --output ./dist/main.js",
-    "build": "webpack --mode production ./src/index.js --output ./dist/main.js"
+    "build": "webpack --mode production ./src/index.js --output ./dist/main.js",
+    "precommit": "NODE_ENV=production lint-staged"
```
Comment (csuriel) [inline package.json]: production mode?

File: .eslintrc (hunk 4 above)
Comment (nathzz30) [reply .eslintrc]: Thank you
i fix it in the last commit

File: package.json (hunk 1 above)
Comment (nathzz30) [reply package.json]: Thank you

File: .gitignore [hunk 6]
```diff
@@ -1,2 +1,3 @@
 node_modules
-dist
\ No newline at end of file
+dist
+package-lock.json
```
Comment (gutierrezf) [inline .gitignore]: add an empty line?

File: package.json (hunk 5 above)
Comment (nathzz30) [reply package.json]: was fixed, now is used webpack-server for production mode

**Comment by nathzz30** [comment]
In this PR i fix this issue:
#1 #2 #4 #5 #6

**Pull request merged**